specification/snapshot/delete_repository/SnapshotDeleteRepositoryRequest.ts

```diff
@@ -22,9 +22,13 @@ import { Names } from '@_types/common'
 import { Duration } from '@_types/Time'
 
 /**
+ * Delete snapshot repositories.
+ * When a repository is unregistered, Elasticsearch only removes the reference to the location where the repository is storing the snapshots.
```

Comment:
```suggestion
 * When a repository is unregistered, Elasticsearch removes only the reference to the location where the repository is storing the snapshots.
```

Reply:
Fixed in https://github.com/elastic/elasticsearch-specification/pull/3314/commits/dc70256ca197d43bd3796691a02dc19e2a9a13f7